flagd/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/cucumber/godog v0.12.6
 	github.com/dimiro1/banner v1.1.0
 	github.com/mattn/go-colorable v0.1.13
+	github.com/open-feature/flagd/core v0.4.4
```

Comment:
This should not be here since we're using workspaces. 
Maybe we should not having the `go.work` files in gitignore 🤔 what's your opinion @james-milligan ?

Reply:
I would still have expected the `go.mod` to contain `flagd/core`, all the go workspace is doing is handling `replace directives` (`=>`) under the covers

If the project is opened at a single module level, i.e. in `./flagd` there should be no issues / errors.

Reply:
@thisthat